packages/web-integration/src/Page.ts

```diff
@@ -0,0 +1,45 @@
+import type { WebKeyInput } from './common/page';
+import type { ElementInfo } from './extractor/extractor';
+
+type encodingType = 'base64' | 'binary';
+type imageType = 'jpeg' | 'png';
+
+export type screenshotOptions = {
+  path?: string;
+  encoding?: encodingType;
+  type?: imageType;
+  quality?: number;
+};
+export type MouseButton = 'left' | 'right' | 'middle';
+
+export abstract class AbstractPage {
+  abstract screenshot(options: screenshotOptions): Promise<Uint8Array>;
+  abstract getElementInfos(): Promise<ElementInfo[]>;
+  abstract url(): string;
+
+  get mouse() {
+    return {
+      click: async (
+        x: number,
+        y: number,
+        options: { button: MouseButton },
+      ) => {},
+      wheel: async (deltaX: number, deltaY: number) => {},
+      move: async (x: number, y: number) => {},
+    };
+  }
+
+  get keyboard() {
+    return {
+      type: async (text: string) => {},
+      press: async (key: WebKeyInput) => {},
+    };
+  }
+
+  async selectAll(): Promise<void> {}
```

Comment:
should be `keyboard.selectAll` ?

Reply:
I haven't figured out how to implement slectAll under Appium, because it's possible to use mouse or touch.

If it is a keyboard implementation, I abstract it into keyboard

Reply:
The final implementation is clearInput method